projectional/src/main/java/jetbrains/jetpad/projectional/cell/BaseProjectionalSynchronizer.java

```diff
@@ -66,6 +70,8 @@
   private DeleteHandler myDeleteHandler = DeleteHandler.EMPTY;
   private ContentKind<SourceItemT> myItemKind;
   private Function<SourceItemT, SourceItemT> myCloner;
+  private ListMap<ContentKind, ContentToItemMapper> myCompatibleContentKinds = new ListMap<>();
```

Comment:
Because it's not "my content kind", it's in fact "outside content kind which we can map to my content kind"